src/saving/SaveManagerGUI.cs

```diff
@@ -106,7 +112,11 @@ public override void _Process(float delta)
         totalSaveSize.Text =
             Math.Round((float)info.diskSpace / Constants.MEBIBYTE, 2).ToString(CultureInfo.CurrentCulture) + " MiB";
 
+        currentAutoSaveCount = SaveHelper.CountSaves("auto_save").count;
```

Comment:
Have you tested if this is too slow with like 100 save files?
If this causes lag then it would feel better if this was computed in a background thread.